File: chatwoot/typescript-node/src/GetAccountCustomAttributeExample.ts
import * as fs from 'fs';
import api from "chatwoot_client"
import models from "chatwoot_client"

const apiCaller = new api.CustomAttributesApi();
apiCaller.setApiKey(api.CustomAttributesApiApiKeys.userApiKey, "USER_API_KEY");

apiCaller.getAccountCustomAttribute(
  0, // accountId
  "0", // attributeModel
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling CustomAttributesApi#getAccountCustomAttribute:");
  console.log(error.body);
});

File: chatwoot/kotlin/src/main/kotlin/oseg/chatwoot_examples/GetAccountCustomAttributeExample.kt
package oseg.chatwoot_examples

import com.chatwoot.client.infrastructure.*
import com.chatwoot.client.apis.*
import com.chatwoot.client.models.*

import java.io.File
import java.time.LocalDate
import java.time.OffsetDateTime
import kotlin.collections.ArrayList
import kotlin.collections.List
import kotlin.collections.Map
import com.squareup.moshi.adapter

@ExperimentalStdlibApi
class GetAccountCustomAttributeExample
{
    fun getAccountCustomAttribute()
    {
        ApiClient.apiKey["api_access_token"] = "USER_API_KEY"

        try
        {
            val response = CustomAttributesApi().getAccountCustomAttribute(
                accountId = 0,
                attributeModel = CustomAttributesApi.AttributeModelGetAccountCustomAttribute.0,
            )

            println(response)
        } catch (e: ClientException) {
            println("4xx response calling CustomAttributesApi#getAccountCustomAttribute")
            e.printStackTrace()
        } catch (e: ServerException) {
            println("5xx response calling CustomAttributesApi#getAccountCustomAttribute")
            e.printStackTrace()
        }
    }
}
